recipes/json_dto/all/conanfile.py

```diff
@@ -59,7 +59,7 @@ def package(self):
         copy(self, pattern="LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
         copy(
             self,
-            pattern="pub.hpp",
+            pattern="*.hpp",
```

Comment:
Please update the test package with another header so we can avoid this in the future 🙏